robotiq_s_model_articulated_gazebo_plugins/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(robotiq_s_model_articulated_gazebo_plugins)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS std_msgs gazebo_plugins actionlib tf image_transport control_msgs trajectory_msgs geometry_msgs sensor_msgs roscpp gazebo_ros robotiq_s_model_articulated_msgs)
+
+find_package(gazebo REQUIRED)
+
+## System dependencies are found with CMake's conventions
+# find_package(Boost REQUIRED COMPONENTS system)
+include_directories(
+  ${PROJECT_SOURCE_DIR}/include
```

Comment:
Just curious: is `${PROJECT_SOURCE_DIR}` really needed? All examples/tutorials I know of simply use `include` (relative to the current `CMakeLists.txt`).